tests/failpoints/cases/test_rawkv.rs

```diff
@@ -178,6 +182,7 @@ fn test_leader_transfer() {
 
 /// Verify correctness on region merge.
 /// See issue #12680.
+#[ignore]
```

Comment:
Why `ignore`?

Reply:
We use enum_dispatch in `causal_ts`, which bring a dependence of `test_raftstore::TestPdClient`. However, `test_raftstore` depend on `causal_ts`. I will break the circular dependency in another PR. After that PR done, i will enable these two cases.

Reply:
Test cases are added back as PR https://github.com/tikv/tikv/pull/13453 is merged.